### PR TITLE
pythonPackages.pynanoleaf: init at 0.0.5

### PR DESCRIPTION
**Pull request by Oro**
###### Motivation for this change
Nanoleaf connectivity for Home Assistant

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by wamserma — APPROVED** [review]
the expression seams reasonable, didn't test though

File: pkgs/development/python-modules/pynanoleaf/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    inherit pname version;
+    sha256 = "2ced000e3c37f4e2ce0ea177d924af71c97007de9e4fd0ef37dcd7b4a6d1b622";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ requests ];
+
+  meta = with lib; {
+    homepage = https://github.com/Oro/pynanoleaf;
```
Comment (jonringer) [inline pkgs/development/python-modules/pynanoleaf/default.nix]: rfc 45
```suggestion
    homepage = "https://github.com/Oro/pynanoleaf";
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

```
[4 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/82962
2 package built:
python37Packages.pynanoleaf python38Packages.pynanoleaf
```

File: pkgs/development/python-modules/pynanoleaf/default.nix (hunk 1 above)
Comment (Oro) [reply pkgs/development/python-modules/pynanoleaf/default.nix]: Thanks for the review! Force pushed the suggested change

**Review by jonringer — COMMENTED** [review]
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

**Comment by Oro** [comment]
pynanoleaf does not have any unit tests. Force pushed skipping tests and pythonImportsCheck, hope that this is the correct way of handling this! Thanks again for the review

(sidenote: there was a dangling whitespace which presumably got included unintentionally by me after running parse-requirements.py for home-assistant, I also force pushed a clean parse-requirements run here without the dangling whitespace)

**Comment by jonringer** [comment]
> pynanoleaf does not have any unit tests. Force pushed skipping tests and pythonImportsCheck, hope that this is the correct way of handling this! Thanks again for the review

Yes, rebasing and force pushing is the correct way to handle this

**Comment by jonringer** [comment]
@GrahamcOfBorg build python3Packages.pynanoleaf

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM

```
https://github.com/NixOS/nixpkgs/pull/82962
2 package built:
python37Packages.pynanoleaf python38Packages.pynanoleaf
```

**Comment by jonringer** [comment]
Congrats @Oro  on your first nixpkgs PR :)

**Pull request merged**